dev/com.ibm.ws.microprofile.graphql.1.0/resources/com/ibm/ws/microprofile/graphql/resources/MPGraphQL.nlsprops

```diff
@@ -0,0 +1,30 @@
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWMGQ
+#COMPONENTNAMEFOR CWMGQ MicroProfile GraphQL API 
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#ISMESSAGEFILE true
+# #########################################################################
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# This file follows the WebSphere Message Guidelines.
+# For more information, visit: 
+# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
+#
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Messages used by MP GraphQL Impl -  range is 0000-2999, plus 9999 for the emergency message
+#-----------------------------------------------------------------------------------------------------------------------------
+
+ERROR_GENERATING_SCHEMA_CWMGQ0001E=CWMGQ0001E: An error was detected that prevents the MicroProfile GraphQL application from starting in web module, {0}.
```

Comment:
In this sentence
`An error was detected that prevents the MicroProfile GraphQL application from starting in web module, {0}.`

Was the error detected in MicroProfile GraphQL or web module {0}? Or do we know?

Reply:
The error is most likely in the web module, though it is possible that a bug in MP GraphQL could cause an exception that would result in this message being printed. But the intention of this message is to indicate an error in the web module.

Reply:
ok- in that case change to:

`An error in the {0} web module  prevented the MicroProfile GraphQL application from starting.`

Reply:
That sounds better - thanks for the suggestion.  I've incorporated it in the latest commit.